libs/core/uagents_core/config.py

```diff
@@ -0,0 +1,6 @@
+
+
+DEFAULT_AGENTVERSE_URL = "https://agentverse.ai"
+DEFAULT_ALMANAC_API_PATH = "/v1/almanac"
+DEFAULT_MAILBOX_API_PATH = "/v1/agents"
```

Comment:
Mailbox API includes several other paths as well. Maybe something like this would be more specific?
```suggestion
DEFAULT_REGISTRATION_PATH = "/v1/agents"
```